dockstore-client/src/test/java/io/dockstore/client/cli/LaunchTestIT.java

```diff
@@ -438,7 +438,7 @@ public void wdlNoTask() throws IOException{
         WorkflowClient workflowClient = new WorkflowClient(api, usersApi, client);
         workflowClient.checkEntryFile(file.getAbsolutePath(), args, null);
 
-        Assert.assertTrue("output should include a successful cromwell run",systemOutRule.getLog().contains("Missing 'task' in WDL file."));
+        Assert.assertTrue("output should include a successful cromwell run",systemOutRule.getLog().contains("Required fields that are missing from WDL file : 'task'"));
```

Comment:
The error message here still doesn't seem to make sense. The test expects to exit since the WDL file is missing fields. But the message says that the output should include a successful run?